app/views/devise/unlocks/new.html.haml

```diff
@@ -3,19 +3,12 @@
 
 = render partial: 'layouts/header', locals: {page_heading: t('.page_heading')}
 
-= simple_form_for(resource, as: resource_name, url: unlock_path(resource_name), html: { method: :post }) do |f|
-  - if resource.errors.any?
-    .validation-summary
-      %ul
-        - resource.errors.each do |attribute, message|
-          %li
-            = "#{attribute.to_s.humanize} #{message}"
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    = form_for(resource, as: resource_name, url: unlock_path(resource_name), html: { method: :post }, builder: GOVUKDesignSystemFormBuilder::FormBuilder) do |f|
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3698/files#r565935858)